src/app/App.js

```diff
@@ -17,6 +21,23 @@ export const App = () => {
         genres={genreList}
         onSelect={handleChangeSelectedGenre}
       />
+
+      <MovieTitle
+        imageUrl={movieTitleData.imageUrl}
+        movieName={movieTitleData.movieName}
+        releaseYear={movieTitleData.releaseYear}
+        genres={movieTitleData.genres}
+        onSelect={handleSelectMovieTitile}
+      />
+
+       <MovieDetail
+        imageUrl={movieDetailData.imageUrl}
```

Comment:
let's pass one property (e.g.`movieInfo`) that contains all these props instead of passing 6 props

Reply:
changed to props